docs/otel/sampling.md

```diff
@@ -0,0 +1,101 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Sampling
+--->
+
+# Semantic Conventions for Sampling
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- toc -->
+<!-- tocstop -->
+
+These attributes reflect the effect of sampling in a telemetry
+collection pipeline.  These attributes describe how items of telemetry
+was collected, making it possible to define Span-to-Metrics pipelines
+and Logs-to-Metrics pipelines, which accurately count items of
+telemetry, despite sampling.
+
+These attributes MAY be modified by components in a collection pipeline
+to convey the amount of sampling that has been carried out for a
+particular item of telemetry.
+
+## Probability sampling
+
+For more information on how to perform and interpret probability
+sampling based on these attributes, [consult OTEP 235](OTEP235).
+
+### Sampling threshold
+
+The OpenTelemetry sampling decision is defined in terms in terms of a
+threshold and a source of randomness.  The threshold expresses how
+many traces out of `1<<56` (i.e., `math.Pow(2, 56)`) are rejected by
+the sampler, and forms the basis of a rejection test defined by `T <=
+R`.  An item is sampled when the item's sampling threshold is less or
+equal to its randomness.
+
+When determining sampling randomness from a span, implementations
```

Comment:
`sampling.randomness` attribute is defined below, but here we say it's coming from threshold or tracestate - is that right?